iridauploader/tests_integration/test_api_integration.py

```diff
@@ -84,9 +84,10 @@ def test_send_and_get_sample(self):
         sample_desc = "test_sample_desc"
         sample = model.Sample(sample_name, sample_desc)
 
-        sample_json_res = self.test_api.send_sample(sample, project_identifier)
+        sample_id = self.test_api.send_sample(sample, project_identifier)
```

Comment:
Wouldn't it make more sense to keep the json response as before? I see below you have to fetch the sample details again which would now be 2 api calls instead of 1.

Reply:
True. I've changed it and modified the test so that the returned values from send_sample are verified, as well as refetching it from the server and checking it again so we know both functions are working as intended.